packages/cli/src/constructs/api-check.ts

```diff
@@ -356,8 +356,12 @@ export class ApiCheck extends Check {
 
     const deps: ScriptDependency[] = []
     for (const { filePath, content } of parsed.dependencies) {
+      const relativePath = pathToPosix(path.relative(Session.basePath!, filePath))
+      if (relativePath.startsWith('..')) {
+        throw new Error(`You cannot reference a dependency file above the Checkly config location folder. Please, modify "${relativePath}" path.`)
```

Comment:
For the error message, what do you think about:

> Failed to parse dependencies for check ${checkLogicalId} "${checkName}". All check dependencies must be contained in the project directory, which is the directory containing the Checkly config file. Checks cannot reference files which are outside of this directory.
>
> Project directory: ${Session.basePath}
> Dependency: ${filePath}

I tried to add the check logicalId/name to give a hint where the issue is coming from. I also think logging the project directory could let the user know where they're allowed to put files.

I think that it could make sense to use the absolute path rather than `relativePath`. I'm not sure if we need to call `path.resolve` on `filePath`.